src/test/java/com/wikia/webdriver/elements/oasis/pages/InfoboxBuilderPage.java

```diff
@@ -122,20 +173,27 @@ public InfoboxBuilderPage addRowComponent() {
     return this;
   }
 
-  public InfoboxBuilderPage addTitleComponent() {
+  public InfoboxBuilderPage addImageComponent() {
     wait.forElementVisible(componentsButtons.get(1));
     componentsButtons.get(1).click();
 
     return this;
   }
 
-  public InfoboxBuilderPage addImageComponent() {
+  public InfoboxBuilderPage addTitleComponent() {
     wait.forElementVisible(componentsButtons.get(2));
     componentsButtons.get(2).click();
 
     return this;
   }
 
+  public InfoboxBuilderPage addHeaderComponent() {
+    wait.forElementVisible(componentsButtons.get(3));
```

Comment:
first store WebElement in variable
then wait for it to be clickable

please correct other occurrences in this file